backend/promptfoo/intent_config.yaml

```diff
@@ -0,0 +1,61 @@
+description: "Intent"
+
+providers:
+  - id: openai:gpt-4o-mini
+    config:
+      temperature: 0
+
+prompts: file://promptfoo_test_runner.py:create_prompt
+
+tests:
+  - description: "questions directed towards the database look ups should have only 1 question -1"
```

Comment:
Should it be 'lookups'?

Reply:
Thanks - forgot that vs code doesn't have spell check, I'm so used to having it
I've run the files both through a spell checker now - thanks for pointing that out